DependencyInjection/Configuration.php

```diff
@@ -688,6 +688,13 @@ private function getOrmCacheDriverNode(string $name): ArrayNodeDefinition
                 ->scalarNode('pool')->end()
             ->end();
 
+        if ($name === 'metadata_cache_driver') {
+            $node->setDeprecated(...$this->getDeprecationMsg(
+                'The "metadata_cache_driver" configuration key is deprecated. PHP Array cache is now automatically registered when %kernel.debug% is false.',
```

Comment:
Why are we deprecating this configuration key? There should be an explanation in the commit message.

Reply:
We are deprecating it because it doesn't make sense to specify any other cache than what we automatically configure now. If we are leaving it, then it would make more sense to change flex recipe instead.